README.md

```diff
@@ -1,55 +1,45 @@
 
 ![image](https://github.com/user-attachments/assets/a93314da-d956-468a-8d9c-59329f747919)
 
-## Design choices
+### `Getting Started`
 
-We are using Vue for its reactivity and lightweightness along side Pinia, a global state management to handle the growing amount of state shared among components. We use Typescript to make the code more readable and more maintainable. Because ROS uses the publisher and subscriber paradigm, we use Roslibjs library to easily interact with ROS in TypesScript, allowing us to use publishers, subscribers, and services. The websocket and the protocol is through Rosbridge Suite
+For an introduction, please refer to the [Getting Started Guide](./docs/getting_started.md).
 
-### Structure of Code:
+For a code-level overview, check out the [Code Overview](./docs/code_overview.md).
 
-- **components**: single file component vue files that contain logic and UI and styling. If the state does not need to be global, put it in the component. Pubs and Subs related to a component should be ran into them
-- **store**: Put all global states here
-  - Use Services for 1-time actions like turn-on-off the camera
-  - Use Sub/Pub when data is continuous like video feed
-- **App.vue**: Pubs and Subs that don't have a component should be ran here
-- **lib**: [Vue Composables](https://vuejs.org/guide/reusability/composables), and any large amount of .ts files goes here
-- **types.ts** Put all global types in here
+If you're working on mission control development, please follow the [installation.\_guide.md](./docs/installation_guide.md)
 
-## Resources & Library Documentation:
+## **Running Mission Control for those not working on Mission Control**
 
-If unfamiliar with Vue and Typescript here are some resources:
+Follow the steps below based on your role and setup:
 
-- [Official Vue Tutorial](https://vuejs.org/tutorial/#step-1)
-- [Typescript Documentation](https://www.typescriptlang.org/docs/handbook/typescript-in-5-minutes-oop.html) -
-- There is not a lot of documentation/tutorial on RosBridge and Roslib library but here is a collection of resources:
-  [RosBridge Protocol](https://github.com/RobotWebTools/rosbridge_suite/blob/ros1/ROSBRIDGE_PROTOCOL.md)
-- Overview: [https://foxglove.dev/images/blog/using-rosbridge-with-ros2/hero.webp](https://foxglove.dev/images/blog/using-rosbridge-with-ros2/hero.webp)
-- [Roslibjs source code]: https://github.com/RobotWebTools/roslibjs
+### **For Software Members not working on mission control**
 
-## Recommended IDE Setup
+- **Testing Locally (on personal Docker):**
+  - Visit [Mission Control](https://www.trickfirerobotics.org/mission-control/) and follow the URC instructions to run the Docker container. Click on the Navbar websocket icon to switch to local development
```

Comment:
I know you acknowledged this, but I don't think we should give incorrect information on our documentation. It will be implemented in the future, but if we tell people to do it now, then it'll get confusing.

Reply:
Looks like the club is switching to Notion for documentation. Will talk at the software meeting about where software documentation for non-software people should go. For now, deleted that part and saved it in a sticky note for later.